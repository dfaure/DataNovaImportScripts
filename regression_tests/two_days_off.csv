#Identifiant;Libellé_du_site;Date_calendrier;Plage_horaire_1;Plage_horaire_2;Plage_horaire_3;Plage_horaire_4;Plage_horaire_5;Plage_horaire_6;Plage_horaire_7;Heure_limite_dépôt_Courrier;Heure_limite_dépôt_Chrono;Heure_limite_dépôt_Colis;Libellé_de_la_fermeture;Bp_Plage_horaire_1;Bp_Plage_horaire_2;Bp_Plage_horaire_3;Bp_Plage_horaire_4;Bp_Plage_horaire_5;Bp_Plage_horaire_6;Bp_Plage_horaire_7;Bp_Libellé_de_la_fermeture
00045A;CHATILLON LA PALUD AP;2020-12-10;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-18;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-02-13;09:00-12:00;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-05;09:00-12:00;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-06;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-21;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-20;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-15;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-03;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-14;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-11-30;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-02;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-04;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-11-19;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-11-24;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-11-23;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-11-22;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-08;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-09;09:00-12:00;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-11-29;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-11-18;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-12;09:00-12:00;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-11-16;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-13;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-18;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-11-17;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-11;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-22;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-07;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-17;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-19;09:00-12:00;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-11-28;09:00-12:00;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-15;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-29;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-13;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-25;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-24;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-08;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-02-03;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-02-11;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-02-06;09:00-12:00;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-02-07;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-26;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-27;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-02-08;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-23;09:00-12:00;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-11-20;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-11-21;09:00-12:00;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-24;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-07;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-02-01;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-12;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-11-25;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-10;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-11-26;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-11;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-17;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-11-27;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-19;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-14;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-31;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-01;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-30;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-16;09:00-12:00;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-02;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-29;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-20;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-03;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-04;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-26;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-01;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-06;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-09;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-05;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-22;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-16;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-28;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-23;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-25;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-30;09:00-12:00;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-31;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-02-09;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-02-16;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-02-05;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-02-12;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-02-04;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-02-15;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-02-02;09:00-12:00;;;;;;;12:00;12:00;12:00;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-28;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-01-21;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2020-12-27;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-02-18;14:00-18:00;;;;;;;14:45;14:45;14:45;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-02-17;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-02-10;FERME;;;;;;;;;;;FERME;;;;;;;
00045A;CHATILLON LA PALUD AP;2021-02-14;FERME;;;;;;;;;;;FERME;;;;;;;
