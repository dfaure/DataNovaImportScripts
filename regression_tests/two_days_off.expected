00045A|CHATILLON LA PALUD AP|Mo,Th 14:00-18:00; Tu,Fr-Sa 09:00-12:00; PH off; 2020 Dec 24,2020 Dec 31,2020 Dec 26,2021 Jan 02 off
